README.md

```diff
@@ -96,30 +98,41 @@ yarn ui:build
 yarn ui:test
 ```
 
-## Configuring the environment
+# Sitemap
+
+The sitemap can be found in <./public/sitemap.xml>
+
+To update its content:
 
-The app has some default configuration, but it's highly encouraged to define your own.
+1. Edit the list of pages in <./src/sitemap.js>
+2. Run `yarn sitemap`
+3. Commit the changes to git
 
-### Local configuration
+# Configuration
+
+The app has some default configuration (in `.env` and `.env.prod` file), but it's highly encouraged to define your own `.env.local`.
 
 Make a copy of `.env` named `.env.local`, this will allow you to set your own configuration only in your local environment.
```

Comment:
```suggestion
Inside `apps/cowswap-frontend`, make a copy of `.env` named `.env.local`, this will allow you to set your own configuration only in your local environment.
```